homeassistant/components/velux/__init__.py

```diff
@@ -0,0 +1,72 @@
+"""
+
+Connects to VELUX KLF 200 interface
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/scene.velux/
+"
+"""
+
+import logging
+import asyncio
+from homeassistant.helpers import discovery
+
+DOMAIN = "velux"
+SUPPORTED_DOMAINS = ['scene']
+_LOGGER = logging.getLogger(__name__)
+VELUX_MODULE = None
+
+REQUIREMENTS = ['pyvlx==0.1.1']
+
+
+@asyncio.coroutine
+def async_setup(hass, config):
+    """Setup device tracker."""
+
+    # pylint: disable=global-statement, import-error
+    global VELUX_MODULE
```

Comment:
Use [`hass.data`](https://home-assistant.io/developers/code_review_component/#componentplatform-communication) to share details between the component and its platforms.